### ISSUE TITLE
Folders or the ability to combine issues

### ISSUE THREAD

**Issue by nightingale-m** [markusfisch/Libra#9]
Some problems have multiple solutions, and each solution has advantages and disadvantages. I would like to be able to combine several issues related to one problem.

**Comment by markusfisch** [comment]
Yes, that makes a lot of sense! I hope I can add something like this soon.
